new/LIT6213MGLions.xml

```diff
@@ -0,0 +1,103 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6213MGLions" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Miracle of St George: Two Samaritans were rescued
+                    from being eaten by lions</title>
```

Comment:
Do we have an issue with the verb tense? @thea-m I think all present or all past, but not a mix of the two, but you tell me :)

Reply:
I later changed all totles into present tense. 
You will see it in the PR I did yesterday. This PR is already merged. But I will incorporate your comments anyway. Thank you!